cypress/e2e/3-testing/login-invalid.cy.js

```diff
@@ -0,0 +1,62 @@
+//The user cannot submit the login form with invalid credentials and is shown a message.
+
+// use pages workflow!!
+
+describe("login, invalid input", () => {
+
+    it("can NOT submit the login form with invalid credentials and is shown a message", () => {
+    
+        cy.visit("https://norofffeu.github.io/social-media-client/")
```

Comment:
```suggestion
        cy.visit("/")
```

The purpose of these tests is to test the work that is in the current branch - not the work that has already been merged and deployed to the internet.

Cypress exposes it's own web server, so you can just load "/"